benchmark/vm/run-in-context.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [1],
+  breakOnSigint: [0, 1],
+  withSigintListener: [0, 1]
+});
+
+const vm = require('vm');
+
+function main(conf) {
+  const n = +conf.n;
+  const options = conf.breakOnSigint ? {breakOnSigint: true} : {};
+  const withSigintListener = !!conf.withSigintListener;
+
+  process.removeAllListeners('SIGINT');
+  if (withSigintListener)
+    process.on('SIGINT', () => {});
+
+  var i = 0;
```

Comment:
Out of curiosity, is there a reason you define `i` here and not in the `for` block where it's used?